instrumentation/nginx/src/script.h

```diff
@@ -34,19 +34,21 @@ struct NgxCompiledScript {
 
 struct ScriptAttributeDeclaration {
   ScriptAttributeDeclaration(
-    opentelemetry::nostd::string_view attribute, opentelemetry::nostd::string_view script)
-    : attribute(ToNgxString(attribute)), script(ToNgxString(script)) {}
+    opentelemetry::nostd::string_view attribute, opentelemetry::nostd::string_view script, bool convertToInt = false)
```

Comment:
Nice catch :smile: 

Would an enum make more sense here (and be a bit more readable when reading the declarations list)?

E.g. 
```
  {"http.scheme", "$scheme"},
  {"net.host.port", "$server_port", ScriptAttributeType_Int},
  {"net.peer.ip", "$remote_addr"},
  {"net.peer.port", "$remote_port", ScriptAttributeType_Int},
```

Then the constructor would be something like
```
ScriptAttributeDeclaration(
    opentelemetry::nostd::string_view attribute, opentelemetry::nostd::string_view script, ScriptAttributeType = ScriptAttributeType_String)
```